test/extended/util/cli.go

```diff
@@ -167,18 +166,24 @@ func (c *CLI) SetupProject(name string, kubeClient kclientset.Interface, _ map[s
 		e2e.Logf("Failed to create a project and namespace %q: %v", c.Namespace(), err)
 		return nil, err
 	}
-	if err := wait.ExponentialBackoff(retry.DefaultBackoff, func() (bool, error) {
-		if _, err := c.KubeClient().Core().Pods(c.Namespace()).List(metav1.ListOptions{}); err != nil {
-			if apierrs.IsForbidden(err) {
-				e2e.Logf("Waiting for user to have access to the namespace")
-				return false, nil
-			}
```

Comment:
as @enj noted, other errors have been ignored here and ended up by being considered success